src/pages/TransactionDetails/components/EventsList/EventExtraData.tsx

```diff
@@ -0,0 +1,26 @@
+import { DataDecode, DecodeMethodEnum } from 'components';
+
+interface EventExtraDataType {
```

Comment:
Should be called `EventExtraDataPropsType`. The `PropsType` suffix helps deducing which interfaces or types are for components and which are for other types of data. 👍

Reply:
Great. 👍